### ISSUE TITLE
iOS - add left edge slide gesture to go back one page

### ISSUE THREAD

**Issue by tomtheall** [ventuso/Fi-Test#75]
In iOS, add the commonly used gesture of sliding from the left edge of the screen to take the user back one level...revealing back to the previous screen.

Not a priority but this will need to be put in, especially helpful on larger devices.

Example 1
1. tapping a pop-up card on the map
2. Network's home page slides from right and takes over.
3. User slides from left edge to return to map.

Example 2
1. tapping "users" on the applet menu
2. quickly going back to network home page by sliding from left edge

Also helpful for navigating menus in the settings pages.

(I don't use android much but if there is an equivalent gesture on the Android OS we should do that to. I assume users just tap the 'back' hard button by habit?)

**Comment by baonguyenventuso** [comment]
@tomtheall 

I don't understand this clearly, can you give an image to explain it. Need to see how that activity is.

**Comment by tomtheall** [comment]
Yes. On iOS go to settings. Tap on an item, then slide your finger from OFF the left hand slide of the phone to ON the phone towards the center. The page will grab and drag with your finger to swipe "back" one page to the root menu you started on. Can also use this gesture to reveal flight menus that come in from the left (like Fi's menu does). Even from draggable maps! This way users don't need to reach up and hit the menu button or back button.

Here is a video I recorded showing 3 examples of the "slide from left edge" gesture in iOS apps.
https://drive.google.com/file/d/0B7ky67jSyQ0NUWVSeU1DSUJOVFU/view?usp=sharing

Example 1 - shows simple Settings menu navigation.
Example 2 - shows a Google Maps left menu reveal from from the map itself.
Example 3 - shows both in combination in GroupMe. Sliding back one level to previous screen, and then revealing the menu flyout.

All sliding finger from off phone (left side) to center of phone. Like a drag gesture that must touch farthest most pixel.
